src_kt/java/arcs/data/TypeVariable.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.data
+
+import arcs.type.Tag
+import arcs.type.Type
+import arcs.type.TypeFactory
+import arcs.type.TypeLiteral
+
+/** [Type] representation of a type variable. */
+class TypeVariable(val variable: TypeVariableInfo)
```

Comment:
I'm not so sure that you'll need this one either. I would have thought that by the time you actually get to the point of interacting with storage, all of the type variables would have been reduced to a concrete type?